streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -272,6 +273,7 @@ void retryWithBackoff(final Map<TaskId, Set<TopicPartition>> tasksToBeCreated, f
                 try {
                     Thread.sleep(backoffTimeMs);
                     backoffTimeMs <<= 1;
+                    backoffTimeMs = Math.min(backoffTimeMs, maxBackoffTimeMs);
```

Comment:
Why this change again? Or could you add this to the description of the PR (it seems not included)?